src/main/CreateAccount.java

```diff
@@ -114,11 +112,12 @@ public void actionPerformed(ActionEvent event) {
 			} else {
 				try {
 					VerifyLogin.saveLogin(username, password);
```

Comment:
Just a note that we'll need to hash the username/password eventually. The simple fix would be to add like 10k to each of the characters so that you can't see it with the visible eye, but it is still "encrypted".